echo/.devcontainer/docker-compose.yml

```
@@ -30,13 +32,28 @@
     image: pgvector/pgvector:0.6.2-pg16
     restart: unless-stopped
     ports:
-      - 5432:5432
+      - 5432:5432      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)